atlasaction/action.go

```diff
@@ -128,7 +136,8 @@ type (
 		SCM         SCM          // SCM is the source control management system.
 		Repo        string       // Repo is the repository name. e.g. "ariga/atlas-action".
 		RepoURL     string       // RepoURL is full URL of the repository. e.g. "https://github.com/ariga/atlas-action".
-		Branch      string       // Branch name.
+		Branch      string       // Currnet Branch name.
+		BaseBranch  string       // BaseBranch is the base branch name.
```

Comment:
I think it's should belong to PullRequest
https://github.com/ariga/atlas-action/pull/331/files#diff-27a1debcaa5772fc7928822261793bc1bd5ebeca936105e889fbc6a9efc93ac5R142

Reply:
I can move it to pull request, but it require big refactor that I dont wanna do in this PR